### DISCUSSION TITLE
Region plugin in Vue giving error while import

### DISCUSSION THREAD

**Discussion by laxmangaian** [katspaugh/wavesurfer.js#3540] [Q&A] [answered]
Hi, I am trying to use **wavesurfer.js** in my application which built using **Vue**. When I import the Region plugin it's showing error like below.

Importing as below:
> import WaveSurfer from 'wavesurfer.js';
> import RegionsPlugin from 'wavesurfer.js@7/dist/plugins/regions.min.js';

Error:
> Could not find module in path: 'wavesurfer.js/dist/plugins/regions.min.js' relative to '/src/components/HelloWorld.vue'

**Comment by katspaugh** [comment]
Try importing it as suggested in the readme:

```js
import Regions from 'wavesurfer.js/dist/plugins/regions.esm.js'
```

**Reply by laxmangaian** [reply]
It is still giving the same error. Please check below image. I am using wavesurfer.js == 7.5.4 version. 

Code Reference: https://codesandbox.io/p/sandbox/holy-shape-vdwnvg?file=%2Fsrc%2Fcomponents%2FHelloWorld.vue%3A57%2C1

![Screenshot from 2024-02-06 12-49-47](https://github.com/katspaugh/wavesurfer.js/assets/148935513/d470a50f-ae73-40b5-83a0-aab6db6a8f36)

**Reply by katspaugh** [reply, marked as answer]
It looks like you're not using NPM, so you need to specify an actual path for the module, or a URL.

E.g. a CDN URL: `import WaveSurfer from 'http://unpkg.com/wavesurfer.js@7.5.6/dist/wavesurfer.esm.js'`

And similarly for the Regions plugin.

**Reply by laxmangaian** [reply]
Thank you it worked fine in my local system after replacing .min to .esm
